docs/content/getting-started/what-is-rerun.md

```diff
@@ -38,6 +38,32 @@ That's a big question for a welcome page. The short answer is that
 Rerun goes to extreme lengths to make handling and visualizing
 multimodal data streams easy and performant.
 
+## What is Rerun for?
+Rerun is built to help you understand complex processes that include rich multimodal data, including 2D, 3D, text, time series, tensors, etc.
+It is used in many industries, including robotics, simulation, computer vision, or anything that involves a lot of sensors.
+Let's look at a more concrete example:
+
+### Example use case
+Say you're building a vacuum cleaning robot and it keeps running into walls. Why is it doing that? You need some tool to debug it, but a normal debugger isn't gonna be helpful. Similarly, just logging text won't be very helpful either. The robot may log "Going through doorway" but that won't explain why it thinks the wall is a door.
+
+What you need is a visual and temporal debugger, that can log all the different representations of the world the robots holds in its little head, such as:
+
+* RGB camera feed
+* depth images
+* lidar scan
+* segmentation image (how the robot interprets what it sees)
+* its 3D map of the apartment
+* all the objects the robot has detected (or thinks it has detected), as 3D shapes in the 3D map
+* its confidence in its prediction
+* etc
+
+You also want to see how all these streams of data evolve over time so you can go back and pinpoint exactly what went wrong, when and why.
+
+Maybe it turns out that a glare from the sun hit one of the sensors in the wrong way, confusing the segmentation network leading to bad object detection. Or maybe it was a bug in the lidar scanning code. Or maybe the robot thought it were somewhere else in the apartment, because its odometry was broken. Or it could be one of a thousand other things. Rerun will help you find out!
```

Comment:
```suggestion
Maybe it turns out that a glare from the sun hit one of the sensors in the wrong way, confusing the segmentation network leading to bad object detection. Or maybe it was a bug in the lidar scanning code. Or maybe the robot thought it was somewhere else in the apartment because its odometry was broken. Or it could be one of a thousand other things. Rerun will help you find out!
```